github_project_tweaks.js

```diff
@@ -15,12 +15,14 @@ function getAllBoardColumns() {
     return document.evaluate(boardColumnXPath, document, null, XPathResult.ORDERED_NODE_ITERATOR_TYPE, null);
 }
 
+// Read the value of the WiP limit from the description field of the board column.
+// If there is no explicit limit, return 0.
```

Comment:
Use [JSDoc style](https://jsdoc.app/about-getting-started.html) comments

Reply:
Will do in separate PR.